Makefile

```diff
@@ -214,14 +215,20 @@ endif
 	@rm -f "$@"
 	ttx "$<"
 
-$(EMOJI).ttf: $(EMOJI).tmpl.ttf $(EMOJI_BUILDER) $(PUA_ADDER) \
+$(EMOJI).ttf: check_coverage $(EMOJI).tmpl.ttf $(EMOJI_BUILDER) $(PUA_ADDER) \
 	$(ALL_COMPRESSED_FILES) | check_vs_adder
 	@$(PYTHON) $(EMOJI_BUILDER) $(SMALL_METRICS) -V $< "$@" "$(COMPRESSED_DIR)/emoji_u"
 	@$(PYTHON) $(PUA_ADDER) "$@" "$@-with-pua"
 	@$(VS_ADDER) -vs 2640 2642 2695 --dstdir '.' -o "$@-with-pua-varsel" "$@-with-pua"
 	@mv "$@-with-pua-varsel" "$@"
 	@rm "$@-with-pua"
 
+check_coverage:
+ifdef CHECK_COVERAGE
+	$(PYTHON) $(CHECK_COVERAGE_PY) -d $(EMOJI_SRC_DIR) -c
+	@echo -n "Proceed building font? [y/N]" && read ans && [ $${ans:-N} = y ]
```

Comment:
why the prompt y/N here?

Reply:
Don't prompt, fail and tell the user what the CLI arg to skip the check is in case they want to try again w/o linting

Reply:
Removed the prompt. It now always checks, or alternatively a user can run 
`make BYPASS_SEQUENCE_CHECK="True` to bypass the checks.